libraries/provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -1,16 +1,44 @@
 ---
 # Remove sync_gateway
 - hosts: sync_gateways:sg_accels
-  become: yes
+
+  vars:
+    couchbase_sync_gateway_package_base_url:
+    couchbase_sync_gateway_package:
+    couchbase_sync_gateway_package_url: "{{ couchbase_sync_gateway_package_base_url }}/{{ couchbase_sync_gateway_package }}"
 
   tasks:
+  - include: tasks/stop-sync-gateway.yml
+    when: ansible_distribution == "CentOS"
+
+  - include: tasks/stop-sync-gateway-windows.yml
+    when: ansible_os_family == "Windows"
```

Comment:
Is it possible to refactor this so that all the windows / centos ops are in a separate playbooks?